nailgun/entities.py

```diff
@@ -4332,6 +4332,25 @@ def bulk_available_incremental_updates(self, synchronous=True, timeout=None, **k
         response = client.post(self.path('bulk/available_incremental_updates'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
 
+    def play_roles(self, synchronous=True, timeout=None, **kwargs):
+        """Play all assigned ansible roles
```

Comment:
As same endpoint for both host and host group, so it'd be nice to cover this func for hostgroup too, wdyt?
```suggestion
        """Play all assigned ansible roles on host
```

Reply:
It would. I did what I needed for my tests, there's a lot of things that would be nice.